src/Http/Client.php

```diff
@@ -217,12 +229,17 @@ public function post(
         int $timeout = self::DEFAULT_TIMEOUT_VALUE
     ) : stdClass {
         $authUsername = $this->getAuthUsername();
-        if (! SecurityClient::validateAuthUser($authUsername)) {
+        if (! SecurityClient::validateAuthUser((string) $authUsername)) {
             throw new HttpException(
                 sprintf('An auth username is required for NS8 %s requests.', self::POST_REQUEST_TYPE)
             );
         }
 
+        $platformIdentifier = $this->getPlatformIdentifier();
+        if (! empty($platformIdentifier)) {
+            $data['platform_uuid'] = $platformIdentifier;
```

Comment:
@crfroehlich This is currently what would be passed in with all requests including events & actions to represent the unique platform identifier. Please let me know if you'd prefer a different name or naming convention.